labelparse.py

```diff
@@ -37,19 +33,19 @@ def repo_fetch():
 
     for repo in dtk_repo_names:
         response = requests.get(
-            'https://api.github.com/repos/newrelic/' + str(repo) + '/issues')
-        if response.status_code == 200:
+            f'https://api.github.com/repos/newrelic/{repo}/issues'
+        )
+        if response:
```

Comment:
This makes use of the `request` library's internal comparator. You don't have to specify the 200 status code.